### ISSUE TITLE
NVDA does not announce edge of table when moving to the right in a table when in IE

### ISSUE THREAD

**Issue by btman16** [nvaccess/nvda#14329]
### Steps to reproduce:
1. Open IE with NVDA 2022.4beta2.
2. Navigate to a site with a table, or a local HTML file with a table (the NVDA What's new file will work).
3. Press T to locate the table.
4. Use ctrl+alt+right arrow to navigate across the row.
### Actual behavior:

NVDA does not say "edge of table" when I hit the right boundary, but it does say this when I hit the left boundary.
### Expected behavior:

In both cases, NVDA should speak "edge of table" when I hit either end.
### NVDA logs, crash dumps and other attachments:
The relevant portion of the log is:
ERROR - scriptHandler.executeScript (19:27:19.437) - MainThread (5488):
error executing script: <bound method DocumentWithTableNavigation.script_nextColumn of <virtualBuffers.MSHTML.MSHTML object at 0x00B59F90>> with gesture 'alt+ctrl+right arrow'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 289, in executeScript
  File "documentBase.pyc", line 480, in script_nextColumn
  File "documentBase.pyc", line 417, in _tableMovementScriptHelper
  File "documentBase.pyc", line 362, in _tableFindNewCell
  File "virtualBuffers\__init__.pyc", line 674, in _getNearestTableCell
TypeError: cannot unpack non-iterable _TableCell object

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2022.4beta2 (2022.4.0.26839)
#### Windows version:
Windows 10
#### Name and version of other software in use when reproducing the issue:
IE 11 is where this issue occurs. It does not occur in Chromium browsers.
#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
NVDA 2022.3.1 works correctly, and this issue does not occur.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

**Comment by seanbudd** [comment]
This has been fixed via #14298, and will be fixed in 2022.4beta3
